packages/node/test/integration/install-virtual.spec.ts

```diff
@@ -93,6 +93,12 @@ describe("Node method follows spec - proposeInstallVirtual", () => {
       "Virtual AppInstance with Node C. All Nodes confirm receipt of proposal",
     () => {
       it("sends proposal with non-null initial state", async done => {
+        console.log("For everyone's benefit from here on out, know this", {
```

Comment:
Is this log needed or was this just for debugging purposes?

Reply:
Debugging, will remove